### PR TITLE
Added -S switch to automate syncing master repository

### PR DESCRIPTION
**Pull request by potiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -343,6 +343,8 @@ Automated checkout of airflow project:
 -B, --branch [BRANCH]
         Branch to check out when cloning the repository specified by -R. [master]
 
+-S, --synchronise-master
+        Synchronizes master of your local and origin remote with the main apache repository.
```
Comment (sprzedwojski) [inline README.md]:
```suggestion
        Synchronizes master of your local and origin remote with the main Apache repository.
```

File: run_environment.sh [hunk 2]
```diff
@@ -48,13 +50,13 @@ CLEANUP_IMAGE=false
 LIST_KEYS=false
 # initializes local virtualenv
 INITIALIZE_LOCAL_VIRTUALENV=false
-# Repository which is used to clone airflow from - wh
-# en it's not yet checked out (default is the Apache one)
+# sync master
+SYNC_MASTER=false
+# Repository which is used to clone airflow from - when
```
Comment (sprzedwojski) [inline run_environment.sh]:
```suggestion
# Repository which is used to clone Airflow from - when
```

File: run_environment.sh [hunk 3]
```diff
@@ -307,6 +309,8 @@ Automated checkout of airflow project:
 -B, --branch [BRANCH]
         Branch to check out when cloning the repository specified by -R. [master]
 
+-S, --synchronise-master
+        Synchronizes master of your local and origin remote with the main apache repository.
```
Comment (sprzedwojski) [inline run_environment.sh:313]:
```suggestion
        Synchronizes master of your local and origin remote with the main Apache repository.
```

File: run_environment.sh [hunk 4]
```diff
@@ -768,6 +774,25 @@ if [[ ${LIST_KEYS} == "true" ]]; then
...
+        pushd ${AIRFLOW_BREEZE_AIRFLOW_DIR} || exit 1
+        set +e
+        git ls-remote --exit-code apache 2>/dev/null >/dev/null
+        if [[ $? != 0 ]]; then
+            echo "Adding remote apache ${APACHE_AIRFLOW_REPO_GIT}"
+            git remote add apache "${APACHE_AIRFLOW_REPO_GIT}"
+        fi
+        echo "Fetching apache repository"
+        git fetch apache
+        echo "Move local master to remote apache master"
```
Comment (sprzedwojski) [inline run_environment.sh]: Maybe a bit more straightforward would be something like:
"Replace local master with remote apache master"

**Review by sprzedwojski — APPROVED** [review]
LGTM, minor grammatical comments

File: run_environment.sh (hunk 4 above)
Comment (potiuk) [reply run_environment.sh]: I changed to "Force local master to be the same as remote apache master"

**Pull request merged**